doc/ref/pres.xml

```diff
@@ -88,8 +88,8 @@ numbering of generators in a family of associative words.
 <Section Label="Printing Presentations">
 <Heading>Printing Presentations</Heading>
 
-Whenever you create a presentation <M>P</M>, say, or assign it to a variable,
-&GAP; will respond by printing <M>P</M>.
+Whenever you create a presentation, let us call it <M>P</M>, or assign it to a
```

Comment:
I think basically removing the “say” is the best option here. Especially as it is the beginning of a section so there is not way that someone could think that P is a pre-existing variable.
```suggestion
Whenever you create a presentation <M>P</M>, or assign <M>P</M> to a variable,
```